.travis.yml

```diff
@@ -5,3 +5,7 @@ services:
 
 script:
   - make build test
+
+after_success:
+  - make coverage
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Not sure it's a good idea to pipe to bash. There's a node codecov client that could be added as a dev dependency and called in the coverage script? Like this: https://github.com/codecov/example-node#nyc.